.github/workflows/flowzone.yml

```
@@ -0,0 +1,45 @@
+name: Flowzone
+
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+    branches:
+      - "main"
+      - "master"
+
+jobs:
+  flowzone-staging:
+    name: balena-staging
+    uses: product-os/flowzone/.github/workflows/flowzone.yml@protect
+    secrets:
+      FLOWZONE_TOKEN: ${{ secrets.FLOWZONE_TOKEN }}
+      GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
+      GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+      BALENA_API_KEY: ${{ secrets.BALENA_STAGING_TOKEN }}
+    with:
+      balena_environment: balena-staging.com
+      balena_slugs: |
+        balena_os/aarch64-supervisor,
+        balena_os/amd64-supervisor,
+        balena_os/armv7hf-supervisor,
+        balena_os/i386-supervisor,
+        balena_os/rpi-supervisor
+      job_name: balena-staging
+
+  flowzone:
+    name: balena-cloud
+    uses: product-os/flowzone/.github/workflows/flowzone.yml@protect
```

This needs to go back to
```suggestion
    uses: product-os/flowzone/.github/workflows/flowzone.yml@master
```
once working and https://github.com/product-os/flowzone/pull/208 is merged